### ISSUE TITLE
DC4: Add Insights template

### ISSUE THREAD

**Issue by obriensystems** [aws-quickstart/quickstart-uipath-orchestrator#36]

**Comment by ubikusss** [comment]
So the request is to add UiPath Insights as a template for AWS?

**Comment by obriensystems** [comment]
Yes, if you have a cf template for the following - already upgraded orch to 21.10.0
we will try following the procedure manually for now via
https://docs.uipath.com/installation-and-upgrade/docs/insights-hardware-and-software-requirements

**Comment by obriensystems** [comment]
manual insights notes
- create a vm with 8vcores/32g win 2019server

Ongoing requirements
- separate RDS for sqlserver for insights

**Comment by ubikusss** [comment]
Ok, thank you!
